app/src/main/java/io/github/sheasmith/discountdetective/repository/ShoppingListRepository.kt

```diff
@@ -42,4 +42,14 @@ class ShoppingListRepository @Inject constructor(
     suspend fun deleteFromShoppingList(shoppingListShoppingListItem: ShoppingListItem) {
         shoppingListDao.delete(shoppingListShoppingListItem)
     }
+
+    suspend fun updateChecked(shoppingListShoppingListItem: ShoppingListItem) {
```

Comment:
Make sure you add a comment to this.

Also you can make this a generic update method as well. Just use the generic Dao method when you've done that.